asv_bench/benchmarks/io/parsers.py

```diff
@@ -1,7 +1,10 @@
 import numpy as np
 
-from pandas._libs.tslibs.parsing import (
-    _concat_date_cols, _does_string_look_like_datetime)
+try:
+    from pandas._libs.tslibs.parsing import (
```

Comment:
What was failing here? I don't think passing on exception is desired

Reply:
The import fails with Pandas 0.23. The import errors are ignored because the benchmark suite is supposed to be importable also on historical pandas releases.

Reply:
Right but I don't think full compatibility with old versions is a goal for the benchmarking suite; that would be a nightmare to maintain! If it's not currently in master then OK to remove

Reply:
It is currently in master.

The point is not to maintain full compatibility of the benchmark suite, but that the failures are not allowed to occur at import time.

Reply:
So IIUC you are saying this prevents any benchmark in the module from being run for historical versions where these imports aren't present?

Reply:
Yes, it prevents running any of the benchmarks if the import fails.

That's of course maybe not optimal, it could be better if import failure disabled only the benchmarks per module, but that's how current versions of asv work.

Reply:
Hmm still a little unsure about this. What criteria would you use to judge when to do this or not?

Reply:
I would trust @pv on this :)

Reply:
I would decide which pandas versions are still important to benchmark against, and then check the benchmark suite is importable with the oldest one. Here, I thought 0.23 is recent enough to warrant some adjustments.

Reply:
Gotcha. Can you just add a comment explaining why we do this then? Ideally this wouldn't prevent the entire suite from running so I think comment would be a good indicator that we should revert if asv gains that robustness in the future

Reply:
Added.